homeassistant/components/fully_kiosk/const.py

```diff
@@ -25,6 +25,10 @@
 
 SERVICE_LOAD_URL = "load_url"
 SERVICE_START_APPLICATION = "start_application"
+SERVICE_SET_CONFIG = "set_config"
```

Comment:
Why a service and not config entities ?

Reply:
It would be hundreds of entities really...

Reply:
There's over 300 settings (per device) that could be changed with this service 😅

Reply:
Okay… a service please 🙏 .